src/reference.rs

```diff
@@ -73,3 +103,32 @@ impl<T> ReferenceOr<Box<T>> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use serde_json::json;
+
+    #[test]
+    fn test_bad_responses() {
+        // Note: missing the "description" field
+        let value = json!({
+            "200": {
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        });
+
+        match serde_json::from_value::<crate::Responses>(value) {
+            Ok(_) => unreachable!(),
+            Err(e) => assert!(
+                e.to_string().contains("missing field `description`"),
+                "unhelpful error: {e}"
```

Comment:
this is not terribly important, but perhaps the assertion failure should be a bit more explicit about what we expect here?
```suggestion
                "error did not indicate what field is missing: {e}"
```